msl_expressions/autogenerated/src/Plans/Behaviours/AlignToGoal.cpp

```diff
@@ -118,7 +119,13 @@ namespace alica
         mc.motion.angle = driveTo->angleTo();
         mc.motion.translation = driveTo->length();
 
-        sendAndUpdatePT(mc);
```

Comment:
This behaviour might drive into the enemy goal if the robot is in the state for too long. Added rule action check.